tlatools/org.lamport.tlatools/src/recomp/RVStrategy.java

```diff
@@ -0,0 +1,51 @@
+package recomp;
+
+import java.util.List;
+
+public class RVStrategy implements Runnable {
+    public static boolean globalIsDone = false;
+
+    private String tla;
+    private String cfg;
+    private String recompType;
+    private String recompFile;
+    private boolean verbose;
+
+    public RVStrategy(String tla, String cfg, String recompType, String recompFile, boolean verbose) throws InterruptedException {
+        this.tla = tla;
+        this.cfg = cfg;
+        this.recompType = recompType;
+        this.recompFile = recompFile;
+        this.verbose = verbose;
+    }
+
+    synchronized static public boolean isThreadDone() {
+        boolean isDone = globalIsDone;
+        globalIsDone = true;
+        return isDone;
+    }
+
+    public static void runRVStrategies(String tla, String cfg, String recompType, String recompFile, boolean verbose) throws InterruptedException {
```

Comment:
I would avoid putting this static method here. Instead, I would put this code directly in RecompVerify.recompVerify() (where you call the method).